common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -402,6 +402,9 @@
                         line = lines[i].trim();
                         if (line.length > 0) {
                             textHint = extractHint(line, true);
+                            if (!textHint.nothint) {
+                                throw new Error("One of the provided options don't have valid text value")
```

Comment:
replace `don't have valid` with `doesn't have a valid`. Are these error messages are marked for internationalization?

Reply:
I have fixed the typo and the messages are already internationalized in the `showErrorMeassage` method.